src/Trace/Span.php

```diff
@@ -0,0 +1,507 @@
+<?php
+/**
+ * Copyright 2017 Google Inc. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+namespace Google\Cloud\Trace;
+
+/**
+ * This plain PHP class represents a Span resource.
+ *
+ * A span represents a single timed event within a Trace. Spans can be nested
+ * and form a trace tree. Often, a trace contains a root span that describes
+ * the end-to-end latency of an operation and, optionally, one or more subspans
+ * for its suboperations. Spans do not need to be contiguous. There may be
+ * gaps between spans in a trace.
+ *
+ * Example:
+ * ```
+ * use Google\Cloud\Trace\TraceClient;
+ *
+ * $traceClient = new TraceClient();
+ * $trace = $traceClient->trace();
+ * $span = $trace->span([
+ *     'name' => 'span name',
+ *     'attributes' => ['foo' => 'bar'],
+ *     'stackTrace' => debug_backtrace(DEBUG_BACKTRACE_IGNORE_ARGS)
+ * ]);
+ * $span->setStartTime();
+ * $span->setEndTime();
+ * ```
+ * @codingStandardsIgnoreStart
+ * @see https://cloud.google.com/trace/docs/reference/v2/rest/v2/projects.traces/batchWrite#Span Span model documentation
+ * @codingStandardsIgnoreEnd
+ */
+class Span implements \JsonSerializable
+{
+    use AttributeTrait;
+    use TimestampTrait;
+
+    /**
+     * @var string [TRACE_ID] is a unique identifier for a trace within a
```

Comment:
Do we need to refer to the traceId as `[TRACE_ID]`?

Reply:
I can change it. It was copied verbatim from the main documentation site.